qiita_pet/static/js/qiita.js

```diff
@@ -117,3 +117,90 @@ function show_hide_process_list() {
     $("#qiita-processing").hide();
   }
 }
+
```

Comment:
it's mainly moving code around so the libraries are available everywhere vs. just in the study listing